src/components/Titles.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class Titles extends Component {
+	render() {
+		return (
+			<div className="menu">
+				<ul className="list-items">
```

Comment:
Your classname is becoming a bit too unsemantic - call it `list-titles` or `notes-titles` instead